oauth2-authorization-server/src/main/java/org/springframework/security/oauth2/server/authorization/web/OAuth2TokenRevocationEndpointFilter.java

```diff
@@ -58,6 +63,7 @@
 	/**
 	 * The default endpoint {@code URI} for token revocation requests.
 	 */
+	protected final Log logger = LogFactory.getLog(this.getClass());
```

Comment:
Please remove logging. Currently, there is no logging in the framework. We'll be adding this in a separate task.

Reply:
Done